single-function/add-two-numbers/README.md

```diff
@@ -0,0 +1,15 @@
+## Prompt
+
+You are given two non-empty linked lists representing two non-negative integers. 
+The digits are stored in reverse order and each of their nodes contain a single digit. 
+Add the two numbers and return it as a linked list.
+
+You may assume the two numbers do not contain any leading zero, except the number 0 itself.
```

Comment:
> except the number 0 itself.
unless the entire number is 0.

(original wording is a bit ambiguous between a "number" and one "digit" of a number).

Reply:
✔️